address-form/src/app/address-form/address-form.component.ts

```diff
@@ -16,39 +16,32 @@ interface Country {
 export class AddressFormComponent{
 
   countries = new FormControl();
-  isCountrySelected: boolean;
   countryList: Country[];
-  listAddresses: Address[];
+  searchResults: any;
 
   addressForm = this.formBuilder.group({
-    streetOne: [''],
-    streetTwo: [''],
+    streetOne: ['', [Validators.required, Validators.minLength(1), Validators.pattern('^[a-zA-Z]+$')]],
```

Comment:
these are the built in validators for street names. I need to create flags for the UI but the form status of valid will change depending on if these are entered.